apps/web/src/pages/api/v1/profile/index.ts

```diff
@@ -42,6 +44,26 @@ handler.post(
       throw new BadRequest('Could not create account')
     }
 
+    // Check if the account is the admin account, and if so set admin permissions
+    if (body.email === Environment.firebaseAdminEmail) {
+      try {
+        const admin = configureAdmin()
+        const firebaseUser = await admin.auth().getUser(request.token.uid)
+        // If user doesn't already have claim, add admin claim
+        if (
+          !firebaseUser.customClaims ||
+          !firebaseUser.customClaims[FirebaseClaim.admin]
+        ) {
+          await admin.auth().setCustomUserClaims(request.token.uid, {
+            [FirebaseClaim.admin]: true,
+          })
+        }
+      } catch (error) {
+        // do nothing if setting claims does not work
+        console.error(error)
```

Comment:
Import `logger` from `@/utils/logger` and then change this line to:

```suggestion
        logger.error(error)
```

Reply:
May also wanna add some additional text to be logged, for example:

```suggestion
        logger.error(error, 'Unable to set claims')
```

Reply:
Aw yes, thanks for catching! I updated with 2de662d.